src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeValidator.java

```diff
@@ -207,16 +207,39 @@ public static void assertContainerMatchesJurisdictionAndPoBox(
         }
     }
 
-    public static void assertClassificationNewApplicationIfPaymentsArePresent(InputEnvelope envelope) {
-        if (envelope.payments != null && !envelope.payments.isEmpty()
-            && envelope.classification != Classification.NEW_APPLICATION) {
+    public static void assertPaymentsEnabledAndAllowedForClassification(
+        InputEnvelope envelope,
```

Comment:
Could we not split these validations ? It would be much easier

Reply:
done